kinetic/distribution.yaml

```diff
@@ -9973,6 +9973,18 @@ repositories:
       url: https://github.com/ros/ros.git
       version: kinetic-devel
     status: maintained
+  ros-pacman:
+    doc:
+      type: git
+      url: https://github.com/carlosquinterop/ros-pacman.git
+      version: 0.0.1
+    release:
+      packages:
+      - pacman
+      tags:
+        release: release/kinetic/{package}/{version}
+      url: https://github.com/carlosquinterop/ros-pacman.git
```

Comment:
You appear to have used the same repository for your source tree and your release repository.

Please separate these into two different repositories. It will work, but the release process creates a lot of content and snapshots that will generally make cloning your repository much less friendly over time. We generally use the pattern `<REPONAME>-release` for the repository name.